types/src/transaction/transaction.rs

```diff
@@ -530,12 +620,35 @@ impl PartialHashing for Transaction {
                     &blake128(tag.get_tag()),
                 ))
             }
+            Transaction::AssetUnwrapCCC {
+                network_id,
+                burn,
+            } => {
+                if !tag.sign_all_inputs || !tag.sign_all_outputs {
```

Comment:
Why does it need to be sign_all_inputs?

Reply:
@joojis

Reply:
I just wanted to disallow creating anothor valid transaction by manipulating the signature tag.
And that's the default value that the SDK creates.